conanfile.py

```diff
@@ -1,39 +1,43 @@
-from conans import ConanFile, CMake, tools
+from conans import ConanFile, CMake
 
 
 class LibgeohashConan(ConanFile):
     name = "Libgeohash"
-    version = "0.1"
-    license = "https://github.com/simplegeo/libgeohash/blob/master/LICENSE"
+    version = "2469d3c"
+    license = "BSD-3"
+    homepage = "https://github.com/simplegeo/libgeohash"
     url = "https://github.com/ebclark2/conan-libgeohash.git"
+    description = "A pure C implementation of the Geohash algorithm. http://geohash.org"
     settings = "os", "compiler", "build_type", "arch"
     options = {"fPIC": [True, False]}
-    default_options = "fPIC=False"
-    exports = ["CMakeLists.txt", "Geohash.cmake", "FindLibgeohash.cmake"]
+    default_options = "fPIC=True"
```

Comment:
Probably the fPIC option has to be completely removed in Windows

Reply:
Ok I'll do it.

@lasote suggested to have a look at some bincrafter's packages and they are doing it in that way so I did it.

See https://github.com/bincrafters/conan-log4cplus/blob/stable/1.2.1/conanfile.py#L36.

Reply:
@memsharded what is the best way to do it? 

Something like that?

```python
def configure(self):
    del self.settings.compiler.libcxx
    if self.settings.os != "Windows":
        del self.options.fPIC
```

Reply:
Yes

Reply:
done